packages/steiger/src/models/config/index.ts

```diff
@@ -48,6 +48,11 @@ export function getEnabledRules() {
   return $enabledRules.getState()
 }
 
+export function getPluginByRuleName(ruleName: string) {
+  // Always returns a plugin because if a rule exists than the plugin provided it must exist too
+  return $plugins.getState().find((plugin) => plugin.ruleDefinitions.some((rule) => rule.name === ruleName))!
```

Comment:
issue: not a big fan of the `!` operator, especially in a case like this. If it does somehow fail to return the plugin, we might have an error that's harder to debug

suggestion: let's throw a descriptive error if the search failed. That way, if it ever happens, we'll know exactly what didn't match our expectations
```suggestion
  const sourcePlugin = $plugins.getState().find((plugin) => plugin.ruleDefinitions.some((rule) => rule.name === ruleName))
  if (sourcePlugin === undefined) {
    throw new Error(`Expected to find a source plugin for the rule "${ruleName}", but didn't`)
  }
  
  return sourcePlugin
```

Reply:
I agree, this is better